Locker/Helpers/devices.json

```diff
@@ -0,0 +1,75 @@
+{
+            "face_id_devices": [
+                "iPhone10,3",
```

Comment:
```suggestion
                "iPhone10,3", // iPhone X Global
```
Can you add names (here and on apiary)?

Reply:
I tried adding comments but JSON doesn't support comments as far as I know. 
https://stackoverflow.com/questions/244777/can-comments-be-used-in-json, there's this workaround: https://www.freecodecamp.org/news/json-comment-example-how-to-comment-in-json-files/

Reply:
Does it make sense to you to expand this model? Maybe to something like this:
```json
{
    "face_id_devices": [
        {
           "id": "iPhone10,3",
           "name": "iPhone X Global"
       },
     ]
}
```

If you think that this will be too much for something that we use at the moment, it's ok to leave it as is :)

Reply:
I believe that adding something like this is will have so many benefits later on. We'll be adding newer models and information like which model is currently contained so we don't have to search for the model numbers instead of just names. I Will add this to both the local JSON file and Apiary.